src/main/java/com/hivemq/client/internal/mqtt/MqttWebSocketConfigImpl.java

```diff
@@ -27,21 +27,28 @@
 public class MqttWebSocketConfigImpl implements MqttWebSocketConfig {
 
     static final @NotNull MqttWebSocketConfigImpl DEFAULT =
-            new MqttWebSocketConfigImpl(DEFAULT_SERVER_PATH, DEFAULT_MQTT_SUBPROTOCOL);
+            new MqttWebSocketConfigImpl(DEFAULT_SERVER_PATH, DEFAULT_QUERY_STRING, DEFAULT_MQTT_SUBPROTOCOL);
 
     private final @NotNull String serverPath;
+    private final @NotNull String queryString;
     private final @NotNull String subprotocol;
 
-    MqttWebSocketConfigImpl(final @NotNull String serverPath, final @NotNull String subprotocol) {
+    MqttWebSocketConfigImpl(final @NotNull String serverPath, final @NotNull String queryString, final @NotNull String subprotocol) {
         this.serverPath = serverPath;
         this.subprotocol = subprotocol;
+        this.queryString = queryString;
```

Comment:
should be the same ordering as the fields itself: `serverPath`, `queryString`, `subprotocol`

Reply:
Done